sql/core/src/test/scala/org/apache/spark/sql/execution/arrow/ArrowConvertersSuite.scala

```diff
@@ -1208,15 +1208,13 @@ class ArrowConvertersSuite extends SharedSparkSession {
     spark.conf.unset(SQLConf.ARROW_EXECUTION_MAX_RECORDS_PER_BATCH.key)
   }
 
-  testQuietly("unsupported types") {
```

Comment:
This test case is modified because:
1. the original test case did not match the intention which was to test `ArrowConverter` with calling and failing `ArrowUtils.toArrowType` but failed earlier in `SchemaReflection` actually.
2. interval is the only type that is unsupported by arrow